includes/ppc-frontend.php

```diff
@@ -0,0 +1,93 @@
+				<p class="description"><?php esc_html_e( 'User will not be able to publish until complete the checklist.', 'pre-publish-checklist' ); ?></p><br>
+				<input type="radio" name="ppc_radio_button_option" value="2" <?php checked( $ppc_radio_button, 2 ); ?> > <div class="ppc_radio_options"><?php esc_html_e( 'Warn User Before Publishing', 'pre-publish-checklist' ); ?></div>
+				<p class="description"><?php esc_html_e( 'A warning message will display on click of the publish button.', 'pre-publish-checklist' ); ?></p><br>
+				<input type="radio" name="ppc_radio_button_option" value="3" <?php checked( $ppc_radio_button, 3 ); ?> > <div class="ppc_radio_options"><?php esc_html_e( 'Do Nothing', 'pre-publish-checklist' ); ?></div>
+				<p class="description"><?php esc_html_e( 'User will be allowed to publish without any warning.', 'pre-publish-checklist' ); ?></p>
+		</td>
+		</tr>
+		<tr>
+			<th scope="row"><p class="ppc-setting-name"><?php esc_html_e( 'Post Types', 'pre-publish-checklist' ); ?></p></th>
+			<td class = "ppc-posttypes">
+	<?php
+	foreach ( get_post_types( $ppc_args, 'objects' ) as $ppc_post_type ) {
+		if ( in_array( $ppc_post_type->labels->name, $ppc_exclude, true ) ) {
+			continue;
+		}
+		if ( 'post' !== $ppc_post_types && post_type_supports( '' . $ppc_post_type->name . '', 'editor' ) === true ) {
+			if ( ! empty( $ppc_post_types ) ) {
+				if ( false !== $ppc_post_types ) {
+					if ( in_array( $ppc_post_type->name, $ppc_post_types, true ) ) {
+						echo '<label for="ForPostType">
+	                     <input type="checkbox" checked name="posts[]" value="' . esc_attr( $ppc_post_type->name ) . '" >
+	                     ' . esc_attr( $ppc_post_type->labels->name ) . '</label><br> ';
+					} else {
+						echo '<label for="ForPostType">
+	                     <input type="checkbox"  name="posts[]" value="' . esc_attr( $ppc_post_type->name ) . '">
+	                     ' . esc_attr( $ppc_post_type->labels->name ) . '</label><br> ';
+					}
+				}
+			} else {
+				echo '<label for="ForPostType">
+	                     <input type="checkbox"  name="posts[]" value="' . esc_attr( $ppc_post_type->name ) . '">
+	                     ' . esc_attr( $ppc_post_type->labels->name ) . '</label><br> ';
+			}
+		} else {
+			if ( 'post' === $ppc_post_type->name && post_type_supports( '' . $ppc_post_type->name . '', 'editor' ) === true ) {
+				echo '<label for="ForPostType">
+	                 <input type="checkbox" checked name="posts[]" value="' . esc_attr( $ppc_post_type->name ) . '">
+	                 ' . esc_attr( $ppc_post_type->labels->name ) . '</label><br> ';
+			}
+			echo '<label for="ForPostType">
+	                 <input type="checkbox"  name="posts[]" value="' . esc_attr( $ppc_post_type->name ) . '">
+	                 ' . esc_attr( $ppc_post_type->labels->name ) . '</label><br> ';
+		}
+	}
+	$ppc_checklist_item_data = get_option( 'ppc_checklist_data' );
```

Comment:
This variable does not seem to be used below this line, also this is defined at the top of the file. Can you check this and remove this line?